Backpack/src/main/java/net/skyscanner/backpack/util/Drawables.kt

```diff
@@ -94,3 +96,14 @@ internal inline fun sizedDrawable(
   override fun getIntrinsicHeight(): Int =
     height
 }
+
+internal fun Drawable.rasterize(
```

Comment:
That's another extensions function. You may see its call in BpkMapMarket.kt file.